packages/client/src/service/request/request.ts

```diff
@@ -18,6 +18,10 @@ const TIMEOUT_DELAY = 5000;
 const HOST = 'https://ya-praktikum.tech/api/v2';
 
 function queryStringify(data: Record<string, string>): string {
```

Comment:
[Исправление]
Если data может быть undefined, нужно передавать параметр как опциональный
```suggestion
function queryStringify(data?: Record<string, string>): string {
```